dex/src/package/doc/logback.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Add debug="true" to debug configuration -->
 <configuration scan="true" scanPeriod="30 seconds">
-    <property name="logback.include.file" value="${logback.include.file:-./logback.xml}"/>
-    <property name="logback.common.pattern" value="${logback.common.pattern:-%date %-5level [%.25thread] %logger{26} - %msg%n%ex}"/>
+    <property name="logback.include.file" value="${logback.include.file:-./logback-user.xml}"/>
```

Comment:
Yet another logback?

Reply:
It's strange to include the file itself :) So, if you want to slightly change the configuration - just add logback-user.xml to the same directory and describe difference in "include" tag.